crank/tests/test_stack.py

```diff
@@ -0,0 +1,188 @@
+"""
+Unit and regression test for the crank package.
+"""
+
+import pytest
+import os, sys, subprocess, filecmp, shutil, json
+import numpy as np
+from crank.DihedralScanner import DihedralScanner, Molecule
+from crank.QMEngine import QMEngine
+from crank.PriorityQueue import PriorityQueue
```

Comment:
This does not appear to be used.

Reply:
Fixed. And several other molecule not used in this tests are taken out, too.